app/build.gradle

```diff
@@ -99,8 +99,6 @@ ext.materialdrawerVersion = '8.4.5'
 
 // if libraries are changed here, they should also be changed in LicenseActivity
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```

Comment:
The Kotlin Plugin automatically adds stdlib dependency to the build, no need to specify it explicitly

Reply:
oh, it's new, right?

Reply:
yes, since a few versions